app/views/scholars_archive/handles/handle_file_404.html.erb

```diff
@@ -0,0 +1,2 @@
+<p> The work you are trying to look at <b><%= @handle_uri %></b> doesn't seem to have the file you requested <b><%= @file  %></b>. If you believe this is an error please notify an administrator. Thank you.</p>
+<p> You can find the works main page here: <a href=<%= @work_path %>> Link to Work </a>
```

Comment:
'works' is missing an apostrophe. Instead of 'Link to Work' maybe display the title of the work? I'm thinking people may not understand some of these terms. So simplifying it to something like "The file requested may be available as part of [Title of Parent Work]"